### ISSUE TITLE
Installation of OwnClound 9.1 fails on Ubuntu 16.04 due to wrong dependencies

### ISSUE THREAD

**Issue by gumpp** [owncloud/core#26999]
On a fresh install of Ubunt 16.04 (Xenial Xerus) I did the following:

```
wget -nv https://download.owncloud.org/download/repositories/9.1/Ubuntu_16.04/Release.key -O Release.key
apt-key add - < Release.key
sh -c "echo 'deb http://download.owncloud.org/download/repositories/9.1/Ubuntu_16.04/ /' > /etc/apt/sources.list.d/owncloud.list"
apt-get update
apt-get install owncloud
```

which resulted in the following error: 
```
Reading package lists... Done
Building dependency tree
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 owncloud : Depends: owncloud-deps-php5 (>= 9.1.3) but it is not going to be installed or
                     owncloud-deps
E: Unable to correct problems, you have held broken packages.

```
Perhaps this is related to issue #25659 ?

**Comment by kdslkdsaldsal** [comment]
When inspecting the https://download.owncloud.org/download/repositories/9.1/Ubuntu_16.04/all/owncloud_9.1.3-1.1_all.deb we can see the following in the control file:

``
Depends: owncloud-files (>= 9.1.3), owncloud-deps-php5 (>= 9.1.3) | owncloud-deps
``

This is obviously wrong / broken and should be:

``
Depends: owncloud-files (>= 9.1.3), owncloud-deps-php5 (>= 9.1.3) | owncloud-deps-php7.0 (>= 9.1.3)
``

**Comment by kdslkdsaldsal** [comment]
cc @crrodriguez @jnweiger

**Comment by crrodriguez** [comment]
Fixed packages (of version 9.1.4) are up now.please report success or failure. 
Thanks
